packages/snaps-controllers/src/snaps/SnapController.test.tsx

```diff
@@ -1845,7 +1845,6 @@ describe('SnapController', () => {
     );
 
     snapController.destroy();
-    await service.terminateAllSnaps();
```

Comment:
How come this is unnecessary?

Reply:
Some of the timing changed and so the Snap is already torn down when we get to the end of the test

Reply:
It was causing the test to fail due to some timing issue where we were trying to terminate a Snap that had already been terminated. Simplest solution was to remove it for now and look into it later